src/OrbitVersion/include/OrbitVersion/OrbitVersion.h

```diff
@@ -9,6 +9,53 @@
 
 namespace orbit_version {
 
+struct Version final {
+  Version() = default;
+  Version(const Version&) = default;
+  Version(Version&&) = default;
+  Version& operator=(const Version&) = default;
+  Version& operator=(Version&&) = default;
+
+  uint32_t major_version;
+  uint32_t minor_version;
+
+  friend bool operator<(const Version& left, const Version& right) {
+    uint64_t left_version = (static_cast<uint64_t>(left.major_version) << 32) | left.minor_version;
+    uint64_t right_version =
+        (static_cast<uint64_t>(right.major_version) << 32) | right.minor_version;
+    return left_version < right_version;
```

Comment:
You could also consider using std::tie which is more readable (at least to me): `return std::tie(left.major_version, left.minor_version) < std::tie(right.major_version, right.minor_version)`

Reply:
done, thanks!